### PR TITLE
Update to Xcode 11.4

### PR DESCRIPTION
**Pull request by SvyatoslavScherbina**

### CODE REVIEW

File: konan/konan.properties [hunk 1]
```diff
@@ -43,20 +43,20 @@ llvmVersion.macos_x64 = 8.0.0
...
+targetSysRoot.macos_x64 = target-sysroot-xcode_11_4-macos_x64
 
 clangFlags.macos_x64 = -cc1 -emit-obj -disable-llvm-passes -x ir
 clangNooptFlags.macos_x64 = -O1
 clangOptFlags.macos_x64 = -O3
 clangDebugFlags.macos_x64 = -O0
 clangDynamicFlags.macos_x64 =
 
-linkerKonanFlags.macos_x64 = -lSystem -lc++ -lobjc -framework Foundation -sdk_version 10.15
+linkerKonanFlags.macos_x64 = -lSystem -lc++ -lobjc -framework Foundation -sdk_version 10.15.4
```
Comment (sbogolepov) [inline konan/konan.properties:59]: Will it work for users, who did not update to the latest macOS?

**Comment by sbogolepov** [comment]
Should `minimalXcodeVersion` and prerequisites in `README.md` be updated as well?

**Comment by SvyatoslavScherbina** [comment]
> Should `minimalXcodeVersion` and prerequisites in `README.md` be updated as well?

I thought about it, but I expect everything to work with Xcode 11.0. Can't actually test though, since I don't have Xcode 11.0 :) Do you?

**Comment by sbogolepov** [comment]
Me neither :(

**Comment by SvyatoslavScherbina** [comment]
Trivial tests work with Xcode 11.2.

**Comment by SvyatoslavScherbina** [comment]
> Should `minimalXcodeVersion` and prerequisites in `README.md` be updated as well?

I've just remembered that `README.md` actually describes building from sources too, so Xcode 11.4 should be mentioned there. See https://github.com/JetBrains/kotlin-native/pull/4067

**Pull request merged**